AppiumForMac/Server/Handlers/AfMHandlers.m

```diff
@@ -695,7 +717,7 @@ - (AppiumMacHTTPJSONResponse *)post_element_value:(NSString*)path data:(NSData*)
 - (AppiumMacHTTPJSONResponse *)post_keys:(NSString*)path data:(NSData*)postData
 {
     return [self executeWebDriverCommandWithPath:path data:postData onMainThread:YES commandBlock:^(AfMSessionController *session, NSDictionary *commandParams, int *statusCode)
-    {
+    {   DDLogCInfo(@"Request:POST /session/:%@/keys",session.sessionId);
```

Comment:
the formatting style is not respected. Also, there always should be a space after comma